Aztec/Classes/NSAttributedString/Conversions/HTMLConverter.swift

```diff
@@ -58,7 +58,22 @@ public class HTMLConverter {
         
         return attributedString
     }
-    
+
+
+    /// Check if the given html string is supported to be parsed into Attributed Strings.
+    ///
+    /// In some cases, like pasting from the Notes app, the generated HTML will have a `<body>` tag, and that
+    /// is not yet supported. In those cases is preferible to abort html parsing.
+    ///
+    /// - Parameter html: The html string to check.
+    /// - Returns: A bool value indicating if the given html string can be handled correctly.
+    ///
+    func isSupported(_ html: String) -> Bool {
+        let processedHTML = pluginManager.process(html: html)
+        let rootNode = htmlToTree.parse(processedHTML)
+        return hasBodyNode(rootNode.children) == false
```

Comment:
Wondering a bit here but... wouldn't it be better to use the elements inside the `body` tag instead of marking the HTML as not-supported?

Reply:
That's an interesting idea!

I believe that if we continue processing HTML, we still have problems when pasting from the Notes app. Basically we will lose all the stylings unless we have a way to take into account the style tags inside the header tag.

I was trying something similar on a first approach to solve this issue and I got stuck on that problem.

Reply:
Then let's move ahead as it is.  I think it's more important to have a working solution right now.  Good call.